CONTRIBUTORS.md

```diff
@@ -249,7 +251,10 @@ GitHub contributors:
 - DogSports
 - Dominic Taylor
 - Dominik Shaim Ulrich
+<<<<<<< HEAD
 - dparis
+=======
+>>>>>>> 8.1.x
```

Comment:
You have a conflict that remains here!

Reply:
Nice catch, fixed here https://github.com/PrestaShop/PrestaShop/pull/36475/commits/48bf66387cf7394ca6738b7c5f73227b67f7867a